composer.json

```diff
@@ -30,8 +30,8 @@
     },
     "require-dev": {
         "composer/package-versions-deprecated": "^1.11",
-        "phpstan/phpstan": "^0.12",
-        "doctrine/coding-standard": "^6.0 || ^8.0",
+        "phpstan/phpstan": "^0.12.84",
```

Comment:
I'd advise pinning the version of static tools, especially now that we started deploying baseline. It will make sure the version used to generate the baseline is the same for everyone, and make sure not to break builds the day before the release :)